aclogview/Form1.cs

```diff
@@ -1068,5 +1067,50 @@ IEnumerable<TreeNode> GetTreeNodes(TreeNodeCollection nodes)
             }
         }
 
+        private void CmdLock_Click(object sender, EventArgs e)
+        {
+            if (CmdLock.Text == "Lock")
+            {
+                CmdLock.Text = "UnLock";
+                listView_Packets.Enabled = false;
+            }
+            else
+            {
+                CmdLock.Text = "Lock";
+                listView_Packets.Enabled = true;
+            }
+                
+        }
+
+        private void cmdforward_Click(object sender, EventArgs e)
+        {
+            if (listView_Packets.SelectedIndices.Count > 0)
+            {
+                int currow = listView_Packets.Items[listView_Packets.SelectedIndices[0]].Index;
+                if (currow++ <= listView_Packets.Items.Count)

```

Comment:
The combination of post-increment and <= here means you're off by two when the conditional is evaluated and you'll get an exception at the end of the list. For clarity I suggest something like:
int nextRow = listView_Packets.Items[listView_Packets.SelectedIndices[0]].Index + 1;
if (nextRow < listView_Packets.Items.Count)